File: LogicaDeProgramacao1e2Bim/2bim1atividade.js
var roleta = Math.floor(Math.random()*6)
var municaolocal = Math.floor(Math.random()*6)

console.log("Roleta: "+roleta+"\nMunição: "+municaolocal)


if (roleta == municaolocal) {
    console.log("💀🎇🔫")   
}
else{
    console.log("Não foi dessa vez!")
}

File: LogicaDeProgramacao1e2Bim/2bim1atividade.py
from random import randint 
from time import sleep

roleta = randint(1,6)
municaolocal = randint(1,6)


if roleta == municaolocal:
    print("💀🎇🔫")
else:
    print("Não foi dessa vez!")
  

municaolocal = randint(1,6)


while True:
    print("Tentando 🔫")
    sleep(2)
    roleta = randint(1,6)
    
    if roleta == municaolocal:
      print("💀🎇🔫")
      break
    else:
        print("Não foi dessa vez!")
